### DISCUSSION TITLE
Help with query sintax

### DISCUSSION THREAD

**Discussion by davevilela** [Nozbe/WatermelonDB#1205] [Q&A]
Hello,
I would like some help to understand how I could achieve the following query without raw querying the database

```const items = await eventCollection.unsafeFetchRecordsWithSQL(
        `SELECT events.id FROM events
          JOIN schedules ON schedules.id = events.parentable_id
          AND events.parentable_type = 'Schedule'
          AND events.status IN ('started', 'draft')
          WHERE schedules.allow_late_inspections IS FALSE`,
      );```

**Comment by KrisLau** [comment]
I think these documents should help: 

- [Querying](https://nozbe.github.io/WatermelonDB/Query.html)
- [AND / OR Nesting]( https://nozbe.github.io/WatermelonDB/Query.html#andor-nesting)
- [Conditions on related tables ("JOIN queries")](https://nozbe.github.io/WatermelonDB/Query.html#conditions-on-related-tables-join-queries)

**Reply by fullstackNRJ** [reply]
database.get<Events>(TableName.Events).query(Q.on(TableName.Schedules,Q.where('allow_late_inspections', false)), Q.where('parentable_type', 'Schedule'), Q.where('status', Q.In(['started', 'draft'])))

The last part events.status in ('started', 'draft') won't work as there's no Q.In operator in watermelon, maybe there's some other way around this.

**Reply by KrisLau** [reply]
@devnrj07 
Please refer to the docs specifically: https://nozbe.github.io/WatermelonDB/Query.html#conditions-with-other-operators WDB has `Q.oneOf`

**Reply by fullstackNRJ** [reply]
@KrisLau oh! right thanks. Idk how I missed it. @davevilela here you go, just use Q.oneOf(['started', 'draft'])

**Reply by KrisLau** [reply]
@devnrj07 No worries, it happens haha 😄
